CONTRIBUTING.md

```diff
@@ -37,6 +37,7 @@ agentkit/
 │       ├── langchain-solana-chatbot/
 │       ├── langchain-twitter-chatbot/
 │       ├── model-context-protocol-smart-wallet-server/
+│       ├── langchain-xmtp-chatbot/
```

Comment:
Apologies, meant to mention these lists are in alphabetical order. Please put the line above `model-context-protocol-smart-wallet-server`